scipy/linalg/lapack.py

```diff
@@ -499,9 +499,43 @@ def get_lapack_funcs(names, arrays=(), dtype=None):
 
     In LAPACK, the naming convention is that all functions start with a
     type prefix, which depends on the type of the principal
-    matrix. These can be one of {'s', 'd', 'c', 'z'} for the numpy
-    types {float32, float64, complex64, complex128} respectevely, and
-    are stored in attribute `typecode` of the returned functions.
+    matrix. These can be one of {`'s'`, `'d'`, `'c'`, `'z'`} for the numpy
```

Comment:
I'd probably just leave it as it was, then. You could double-backtick each one if you really want.